### DISCUSSION TITLE
Environment variable not working with Vite and Vue 3

### DISCUSSION THREAD

**Discussion by Tarkbyte** [vercel/community#2879] [Help]
### Summary

Hello, 
I built an app with Vue 3 and Vite, and deployed it to Vercel, but for some reason, Vercel is not recognizing the environment variable.

This is what I added into my vercel project
![image](https://github.com/vercel/community/assets/58805919/a467ea1a-35e2-482b-951c-21d8fce0bfc1)

And in my `.env` file: `VITE_API_URL=https://jsonplaceholder.typicode.com`

### Steps to Reproduce

Create a Vite project, add an environtment variable and deploy it to Vercel.

**Comment by amyegan** [comment]
Hi @Tarkbyte. Are you using `process.env.VITE_API_URL` or `import.meta.env.VITE_API_URL` to access the [Vite environment variable](https://vitejs.dev/guide/env-and-mode.html)?

**Comment by Tarkbyte** [comment]
Hello, Thanks for your answer.

I am using `import.meta.env.VITE_API_URL`

**Reply by amyegan** [reply]
I wasn't able to reproduce the problem. Can you walk me through the deployment steps so I can try to deploy my project exactly as you do yours?

**Reply by Tarkbyte** [reply]
Hello I just deleted the current project and imported it again. Added the env variable in this process and seems that it is working now.

![image](https://github.com/vercel/community/assets/58805919/62ec4335-df13-4fb1-872a-f0487d817790)

The first time I tried, I didn't have any env variable specified when importing the project, I added it manually later through the settings, maybe that's what caused the problem. Maybe you can take a look?

Thanks for your answers!

**Reply by gideonmaina** [reply]
@Tarkbyte  Any luck so far? I am experiencing the same problem

**Comment by gideonmaina** [comment]
> @Tarkbyte Any luck so far? I am experiencing the same problem

Solved: Redeploying the app after adding environmental variables
